src/Storages/MergeTree/MergeTreeData.cpp

```diff
@@ -937,6 +947,7 @@ void MergeTreeData::loadDataParts(bool skip_sanity_checks)
 
     std::mutex mutex;
 
+    DataPartsVector duplicate_parts_to_remove;
```

Comment:
Worth putting a comment why this is possible? (because some of disk may be broken and then can be recovered)

Reply:
I feel that it's evident. (BTW there might be other cases to encounter duplicated parts)